Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,666 @@
+    return readable_output
+
+
+def activities_to_human_readable(activities, activity_id):
+    if not activity_id:
+        activities_readable_outputs = []
+        for activity in activities:
+            activities_readable_outputs.append(activity_to_human_readable(activity))
+    else:
+        activities_readable_outputs = (activity_to_human_readable(activities))
+    headers = ['activity_id', 'activity_date', 'app_name', 'description', 'severity']
+    human_readable = tableToMarkdown('Microsoft CAS Activities', activities_readable_outputs, headers, removeNull=True)
+    return human_readable
+
+
+def arrange_entity_data(activity):
+    entities_data = []
+    if 'entityData' in activity.keys():
+        entity_data = activity['entityData']
+        if entity_data:
+            for key, value in entity_data.items():
+                if value:
+                    entities_data.append(value)
+            activity['entityData'] = entities_data
+
+
+def arrange_entities_data(activities, activity_id):
+    if not activity_id:
+        for activity in activities:
+            arrange_entity_data(activity)
+    else:
+        arrange_entity_data(activities)
+    return activities
+
+
+def list_activities_command(client, args):
+    url_suffix = '/activities/'
+    activity_id = args.get('activity_id')
+    custom_filter = args.get('custom_filter')
+    arguments = assign_params(skip=args.get('skip'), limit=args.get('limit'), service=args.get('service'),
+                              instance=args.get('instance'), ip=args.get('ip'), ip_category=args.get('ip_category'),
+                              username=args.get('username'), taken_action=args.get('taken_action'),
+                              source=args.get('source'))
+    request_data, url_suffix = build_filter_and_url_to_search_with(url_suffix, custom_filter, arguments, activity_id)
+    activities = client.list_activities(url_suffix, request_data)
+    if activities.get('data'):  # more than one activity
+        activities = activities.get('data')
+    activities = arrange_entities_data(activities, activity_id)
+    human_readable = activities_to_human_readable(activities, activity_id)
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='MicrosoftCloudAppSecurity.Activities',
+        outputs_key_field='_id',
+        outputs=activities
+    )
+
+
+def file_to_human_readable(file):
+    readable_output = assign_params(owner_name=file.get('ownerName'), file_id=file.get('_id'),
+                                    file_type=file.get('fileType'), file_name=file.get('name'),
+                                    file_access_level=file.get('fileAccessLevel'), app_name=file.get('appName'),
+                                    file_status=file.get('fileStatus'))
+    return readable_output
+
+
+def files_to_human_readable(files, file_id):
+    if not file_id:
+        files_readable_outputs = []
+        for file in files:
```

Comment:
why not design it like so - when there's only one file, `files` will be an array with one file?
would simplify the code